### PR TITLE
fix(Events sorting): 20278 fixes for sorting events by date. Added "startedAt" field to EventCard

### PR DESCRIPTION
**Pull request by albaranau**
Additional PR with some fixes for: 
https://kontur.fibery.io/Tasks/Task/FE-Default-sorting-for-GG-app-20278/comment=25023

## Summary by CodeRabbit

- **New Features**
	- Added localization support for the term "Started".
	- Introduced a new class `.timeInfo` for improved layout in the EventCard component.
	- Added a new function for sorting events by date.

- **Improvements**
	- Enhanced time display in the EventCard to show both start and updated times clearly.
	- Streamlined feature flag handling in the EventsPanel for better configuration management.

- **Bug Fixes**
	- Temporarily disabled sorting functionality in the FullState component by commenting out the EventListSortButton.

### CODE REVIEW

**Comment by albaranau** [comment]
@coderabbitai pause

**Review by dqunbp — APPROVED** [review]

File: src/features/events_list/components/EventCard/EventCard.module.css [hunk 1]
```diff
@@ -33,9 +33,15 @@
   display: flex;
   color: var(--text-faint-strong);
   justify-content: space-between;
+  align-items: end;
```
[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/events_list/components/EventCard/EventCard.module.css]: Fixed

File: src/features/events_list/components/EventCard/EventCard.tsx [hunk 2]
```diff
@@ -86,7 +92,10 @@ export function EventCard({
       ) : null}
 
       <div className={s.footer}>
-        <Text type="caption">{i18n.t('updated') + ` ${formattedTime}`}</Text>
+        <div className={s.timeInfo}>
+          <Text type="caption">{i18n.t('started') + ` ${formattedStartedAt}`}</Text>
+          <Text type="caption">{i18n.t('updated') + ` ${formattedUpdatedAt}`}</Text>
+        </div>
```
[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/events_list/components/EventCard/EventCard.tsx]: fixed

**Review by Natallia-Harshunova — APPROVED** [review]

**Comment by albaranau** [comment]
https://github.com/coderabbitai resume

**Comment by albaranau** [comment]
@coderabbitai resume

**Comment by albaranau** [comment]
@coderabbitai pause

File: src/features/events_list/components/EventsPanel/EventsPanel.tsx [hunk 3]
```diff
@@ -80,15 +67,17 @@ export function EventsPanel({
...
-  const [{ data: eventsList, error, loading }] = useAtom(eventListResourceAtom);
 
   const sortedEvents = useMemo(
     () =>
-      eventsList && sortOrder ? sortEventsByDate(eventsList, sortOrder) : eventsList,
-    [eventsList, sortOrder],
+      eventsList && eventsSortingConfig.order
+        ? sortEventsByDate(eventsList, eventsSortingConfig.order)
+        : eventsList,
+    [eventsList, eventsSortingConfig],
```
[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/events_list/components/EventsPanel/EventsPanel.tsx]: fixed

File: src/features/events_list/helpers/sorting.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import type { Event } from '~core/types';
+
+export function sortEventsByDate(events: Event[], order: 'asc' | 'desc'): Event[] {
+  return [...events].sort((a, b) => {
+    const dateA = new Date(a.startedAt).getTime();
+    const dateB = new Date(b.startedAt).getTime();
+    return order === 'desc' ? dateB - dateA : dateA - dateB;
+  });
+}
```
[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/events_list/helpers/sorting.ts]: Fixed

**Pull request merged**